contracts/mock/IrisScoreProviderMock.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.4.24;
+
+import "../interfaces/IrisScoreProviderI.sol";
+
+
+contract IrisScoreProviderMock {
+
+    uint256 public val = 42;
+
+    function report(bytes32 dataHash) public view returns (uint256) {
+        require(dataHash != 0);
+        return val;
+    }
+
+    function setVal(uint256 newVale) public {
+        val = newVale;
+    }
+}
```

Comment:
We could have a mock provider that assigns from outside a different value for each dataHash. Like the setValue function could receive the dataHash and in the report function returns the score for that dataHash.

Reply:
The mock is only for testing.  This is simple and sufficient for the testing.